VaticanBAV/et/BAVet157.xml

```diff
@@ -0,0 +1,233 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BAVet157" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Śǝna fǝṭrat</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS0003BAV"/>
+                        <collection>Aethiopici</collection>
+                        <idno>Aeth. 157</idno>
+                    </msIdentifier>
+                    <msContents>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r" to="76v"/>
+                            <title type="complete" ref="LIT2314senafe"> The Beauty of the Creation</title>
+                            <explicit xml:lang="am">ድል፡ መነሣቱ፤ በመስቀል፡ ነውና፤ ምነው፡ በዕለተ፡ ዓርብ፡ ወረደ፤ ቢሉ። ኋላ፡ ፍጹም፡ ጥፍአቱ፡ በዕለተ፡ ዓርብ፡ ነውና፤ 
+                                ተፈጸመ፡ ስነ፡ ፍጥረት፤ በሰላመ፤ እግዚአብሔር፡ ዘሎቱ፡ ስብሐት፤ ወጸጋሁ፤ ተሀሉ፤ ምስለ<supplied reason="omitted">፡</supplied> ኃጥእ፡</explicit>
+                            <textLang mainLang="am" otherLangs="gez"/>
+                            </msItem>
+                        
+                        <msItem xml:id="ms_i2">
+                            <locus from="77r" to="77v"/>
+                            <title type="incomplete" ref="LIT2839RepCh123">Malkǝʾ-hymn to Jesus </title>
+                            <note>The text ends abruptly with the repetition of the opening words: <q><sic>ሰለም</sic>፡ ለዝክረ<supplied reason="omitted">፡</supplied> ስምከ<supplied reason="omitted">፡</supplied> <choice><corr></corr><sic> ስም</sic></choice><supplied reason="omitted">፡</supplied> መሃላ፡ ዘ<gap reason="omitted"></gap></q></note>
```

Comment:
Here `<corr></corr>` should be corrected into `<corr resp="bm:GrebTiss1935Codices">ስመ</corr>`